### PR TITLE
Add recipe for cx_oracle

### PR DESCRIPTION
**Pull request by lincrosenbach**

### CODE REVIEW

**Comment by beckermr** [comment]
It might be good to look at the defaults recipe too: https://github.com/AnacondaRecipes/aggregate/tree/master/cx_oracle-feedstock/recipe

File: recipes/cx_oracle/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  license_file: LICENSE.txt
+  summary: Python interface to Oracle
+  doc_url: https://cx-oracle.readthedocs.io/
+  dev_url: https://github.com/oracle/python-cx_Oracle
+
+extra:
+  recipe-maintainers:
+    - lincrosenbach
+    - kfasolin
+    - nicoddemus
```
Comment (nicoddemus) [inline recipes/cx_oracle/meta.yaml]: :+1: for being a maintainer

**Comment by beckermr** [comment]
Looks like defaults patched the windows build which might be the error here.

File: recipes/cx_oracle/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{% set name = "cx_Oracle" %}
+{% set version = "6.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f287776f9b24054fe3470751e44355d28122dcc2aceba7d75e68386125459525
```
[the remark this answers is not in the document]
Comment (lincrosenbach) [reply recipes/cx_oracle/meta.yaml]: Thanks. Added.

**Comment by lincrosenbach** [comment]
@conda-forge/staged-recipes @beckermr I think this is ready for merge.

**Pull request merged**